### ISSUE TITLE
Error happens in segmentation task, but fixed!

### ISSUE THREAD

**Issue by zem007** [MrGiovanni/ModelsGenesis#10]
error happens with:
**_output = activate(final_convolution, axis=1)_**
cause its output is not a **keras layer**.
suggest to replace with this line:
**_output = keras.layers.Softmax(axis = 1)(final_convolution)_**
it works!!!

By the way, good job 纵苇! Your pretrained model improves my project performance.

**Comment by MrGiovanni** [comment]
Updated, thank you very much for your help.
